index.md

```diff
@@ -3,15 +3,54 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
 
-<!-- this is an html comment -->
+This tutorial is built to teach you ESMValTool.  
 
-{% comment %} This is a comment in Liquid {% endcomment %}
+> ## What will you learn in this course
+>
+>   - What is ESMValTool
+>
+>   - How to install ESMValTool
+>
+>   - How to run ESMValTool
+>
+>   - How to develop your own diagnostics and recipes
+>
+>   - How to contribute your recipes and diagnostics back into ESMValTool
+>
+{: .checklist}
 
 > ## Prerequisites
 >
-> FIXME
+> Basic understanding of git (optional - but useful)
+> Basic understanding of your preferred command line interface (ie a bash terminal)
+> Access to CMIP data
+> Access to a suitable computing system (egie jasmin)
```

Comment:
Should be either "eg" or "ie"?
And should "jasmin" be renamed "Jasmin"?
Do we want to add any other computing system here that people who do not recognize "Jasmin" might understand?

Reply:
What is the name of DLR's machine?

Reply:
At DLR we run stuff on Mistral at DKRZ.